packages/checkout-storefront/src/providers/AppConfigProvider/AppConfigProvider.tsx

```diff
@@ -21,7 +21,7 @@ export const AppConfigProvider: React.FC<PropsWithChildren<{ env: AppEnv }>> = (
   env,
 }) => {
   const [{ data: storedAppConfig, loading }] = useFetch(getAppConfig, {
-    args: { checkoutApiUrl: env.checkoutApiUrl },
+    args: { checkoutApiUrl: env.checkoutApiUrl, saleorApiDomain: new URL(env.apiUrl).hostname },
```

Comment:
Again - do we want _just_ the domain or domain + port (if different than the default one)?